src/cascadia/TerminalApp/Profile.cpp

```diff
@@ -49,7 +49,13 @@ static const std::wstring CURSORSHAPE_FILLEDBOX{ L"filledBox" };
 static const std::wstring CURSORSHAPE_EMPTYBOX{ L"emptyBox" };
 
 Profile::Profile() :
-    _guid{},
+    Profile(winrt::guid{})

```

Comment:
I don't like this, instantiating the base with an empty GUID to immediately replace it. But I'm going to guess that this was approx the best you could do without more significant gymnastics given that our ID creation functions are out-param-and-result-code based.

Reply:
indeed. Now, I could just make a return-or-throw-exception version and slap that into utils too. fair?

Reply:
that's fine with me.